generator/generator-opc-generator/generators/app/templates/dist/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= componentName %></title>
+    <script src="<%= componentName %>.js"></script>
+    <style>
+        body {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
+        }
+        <%= componentName %> {
+            --op-component--text-size: 4rem;
```

Comment:
@hybridx hey can we rename the variables to follow --op-c--TextSize. checkout the bem structure once

Reply:
Yes i forgot about the bem structure then it should be
```
--opc__TextSize
```